tests/framework/population/test_manager.py

```diff
@@ -187,89 +187,105 @@ def test_get_population_different_attribute_types() -> None:
-    )
-    assert_squeezing_multi_level_single_outer_multi_inner(unsqueezed, squeezed)
-
-    # Multi-level, multiple outer -> full unsqueezed multi-level dataframe
-    unsqueezed = sim._population.get_population(
-        ["test_column_1", "attribute_generating_columns_6_7"], squeeze=False
-    )
-    squeezed = sim._population.get_population(
-        ["test_column_1", "attribute_generating_columns_6_7"], squeeze=True
-    )
-    assert_squeezing_multi_level_multi_outer(unsqueezed, squeezed)
-
-
-def test_get_population_squeezing_all() -> None:
-
-    # Single-level, single-column -> series
-    # The time manager creates the 'simulant_step_size' column always
-    sim = InteractiveContext(setup=True)
-    unsqueezed = sim._population.get_population("all", squeeze=False)
-    squeezed = sim._population.get_population("all", squeeze=True)
-    assert_squeezing_single_level_single_col(unsqueezed, squeezed, "simulant_step_size")
-
-    # Single-level, multiple-column -> dataframe
-    sim = InteractiveContext(components=[ColumnCreator()], setup=True)
-    unsqueezed = sim._population.get_population("all", squeeze=False)
-    squeezed = sim._population.get_population("all", squeeze=True)
-    assert_squeezing_single_level_multi_col(unsqueezed, squeezed)
-
-    # Multi-level, single outer, single inner -> series
-    sim = InteractiveContext(components=[MultiLevelSingleColumnCreator()], setup=True)
-    sim._population._attribute_pipelines.pop("simulant_step_size")
-    unsqueezed = sim._population.get_population("all", squeeze=False)
-    squeezed = sim._population.get_population("all", squeeze=True)
-    assert_squeezing_multi_level_single_outer_single_inner(
-        unsqueezed, squeezed, ("some_attribute", "some_column")
-    )
-
-    # Multi-level, single outer, multiple inner -> inner dataframe
-    sim = InteractiveContext(components=[MultiLevelMultiColumnCreator()], setup=True)
-    sim._population._attribute_pipelines.pop("simulant_step_size")
-    unsqueezed = sim._population.get_population("all", squeeze=False)
-    squeezed = sim._population.get_population("all", squeeze=True)
-    assert_squeezing_multi_level_single_outer_multi_inner(unsqueezed, squeezed)
-
-    # Multi-level, multiple outer -> full unsqueezed multi-level dataframe
-    sim = InteractiveContext(
-        components=[ColumnCreator(), AttributePipelineCreator()], setup=True
-    )
-    unsqueezed = sim._population.get_population("all", squeeze=False)
-    squeezed = sim._population.get_population("all", squeeze=True)
-    assert_squeezing_multi_level_multi_outer(unsqueezed, squeezed)
+class TestGetPopulationSqueezing:
+    """Tests for squeeze behavior on get_population with specific columns."""
+
+    @pytest.fixture
+    def sim(self) -> InteractiveContext:
+        return InteractiveContext(
+            components=[ColumnCreator(), AttributePipelineCreator()], setup=True
+        )
+
+    def assert_squeezing(
```

Comment:
Nit: aren't the two `assert_squeezing()` methods are the same with a default value of "all" for the `columns` argument?